docs/content/posts/open-source-introduction.md

```diff
@@ -0,0 +1,204 @@
+---
+date: 2019-12-26
+title: Open source
```

Comment:
Can't use ":" symbol or escape it in a page title.

How about "Open source for CI/CD and test infrastructure for Android"?

Reply:
"Open source for CI/CD and test infrastructure for Android" sounds great

Reply:
```suggestion
title: Open source for CI/CD and test infrastructure for Android
```